config/configgrpc/configgrpc.go

```diff
@@ -212,8 +212,14 @@ type ServerConfig struct {
 
 // NewDefaultServerConfig returns a new instance of ServerConfig with default values.
 func NewDefaultServerConfig() ServerConfig {
+	netAddr := confignet.NewDefaultAddrConfig()
+
+	// gRPC conventionally uses HTTP/2, which uses TCP.
+	netAddr.Transport = confignet.TransportTypeTCP
```

Comment:
Sorry, but I fail to understand where is this statement coming from? There is a correlation between L7 and L4 that I fail to see..